go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/EmbeddedEnterprises/autobahnkreuz
+
+go 1.12
```

Comment:
Why is go 1.12 needed?

Reply:
I did not touch this file. This is the file, which was generated from `burrow migrate`. My local version is a 1.12, but this should not matter.

Reply:
Okay, could you check if go 1.12 is really needed?